Update omniauth dependecy to 2.0 and version bump.

josefngo opened the pull request:
Others gems such as omniauth-snapchat require a omniauth version of 2.0 which creates a conflict. This update should remove that conflict. Also bumped the bundler version.

josefngo committed:
Update omniauth dependecy to 2.0 and version bump.

Others gems such as omniauth-snapchat requires a omniauth version of 2.0
which creates a conflict. This update should remove that conflict.

kristofferek reviewed (commented):
I think we should remove the .travis.yml file and then you'll have to rename the gem, similar to how I did here https://github.com/burtcorp/omniauth-yahoo-oauth2/pull/1/commits/39749850fa87d0d528f1f7e10aa4d46db3a32b91. Looks like `omniauth-amazon-oauth2` is available.

josefngo committed:
Delete .travis.yml and rename gem.

kristofferek committed:
Remove omniauth dependency

The omniauth-oauth2 already has a dependency on omniauth so it’s better to have it control the omniauth support. There is no point in us limiting it here.
